### PR TITLE
201911 update 1

### PR DESCRIPTION
**Pull request by mugshepherd**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+# Dependency directories
+node_modules/
```
Comment (tomratcliffe) [inline .gitignore]: Could you add a new line at the end of the file here please? You can turn it on in VSCode as well to automatically add it:

image

File: index.html [hunk 2]
```diff
@@ -219,23 +219,22 @@ <h4 class="ibm-padding-bottom-0 ibm-padding-top-0">Application closing soon. Sub
 
                                         <div class="ibm-col-12-4 ibm-col-medium-12-6">
                                             <div class="ibm-card ibm-card--noborder">
-                                                <a href="https://ibm.biz/CIO-London-Full-Stack-Web-Developer" id="Full-Stack-Web-Developer" target="_blank" rel="noopener noreferrer" class="ibm-blocklink ibm-padding-content">
+                                                <a href="https://careers.ibm.com/ShowJob/Id/708719/Full-Stack-Web-Developer/?lang=en" id="Full-Stack-Web-Developer" target="_blank" rel="noopener noreferrer" class="ibm-blocklink ibm-padding-content">
```
Comment (tomratcliffe) [inline index.html]: I've asked the rest of the team if we can get the short URLs updated so these could then be left as is - makes it easier for the recruitment team to keep them updated without requiring as many changes from us!

I'll get back to you on it :+1:

File: index.html [hunk 3]
```diff
@@ -256,26 +255,34 @@ <h3 class="ibm-padding-top-1 ibm-bold">UX Researchers</h3>
...
-                                                        <h3 class="ibm-padding-top-1 ibm-padding-bottom-0 ibm-bold">Business Analyst</h3>
-                                                        <h4 class="ibm-padding-top-0">Roles now filled.</h4>
+                                                    <h3 class="ibm-padding-top-1 ibm-padding-bottom-0 ibm-bold">Business Analyst</h3>
                                                     <p class="ibm-textcolor-blue-80 ibm-light ibm-padding-top-1 ibm-padding-bottom-2">You’re an analytical thinker and negotiator who effectively communicates complex ideas to both technical and non-technical audiences. You maintain your team’s focus on delivering to business results, and are committed to becoming a trusted partner of the product owner.</p>
-
-                                                </div>
+                                                    
+                                                    <p classj="ibm-ind-link"><span class="ibm-forward-link ibm-btn-sec ibm-btn-blue-60">See full job description</span></p>
```
Comment (tomratcliffe) [inline index.html]: You've got a `classj` here instead of `class`

File: index.html [hunk 4]
```diff
@@ -363,7 +370,7 @@ <h4 class="ibm-padding-top-0">Roles now filled.</h4>
                                 <div class="ibm-fluid ibm-padding-top-3 ibm-padding-bottom-2">
 
                                     <div class="ibm-col-12-12 cio-info-grid">
-                                        <div class="cio-info-grid__hero-image cio-image" style="background-image: url(assets/images/team.jpg)">
+                                        <div class="cio-info-grid__hero-image cio-image" style="background-image: url(assets/images/work-space-1.jpg)">
```
Comment (tomratcliffe) [inline index.html]: Could we remove the old images that aren't being used anymore please?

File: package.json [hunk 5]
```diff
@@ -0,0 +1,22 @@
+{
+  "name": "studio-hiring",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
```
Comment (tomratcliffe) [inline package.json]: If you add a line here for starting the server up locally then you won't have to install `http-server` globally :+1:

Something like:
`"start": "http-server -p 9081"`

Then you can just run `npm start`

File: index.html [hunk 6]
```diff
@@ -256,26 +255,34 @@ <h3 class="ibm-padding-top-1 ibm-bold">UX Researchers</h3>
...
                                                     <p class="ibm-textcolor-blue-80 ibm-light ibm-padding-top-1 ibm-padding-bottom-2">You’re an analytical thinker and negotiator who effectively communicates complex ideas to both technical and non-technical audiences. You maintain your team’s focus on delivering to business results, and are committed to becoming a trusted partner of the product owner.</p>
-
-                                                </div>
+                                                    
+                                                    <p classj="ibm-ind-link"><span class="ibm-forward-link ibm-btn-sec ibm-btn-blue-60">See full job description</span></p>
+                                                </a>
                                             </div>
                                         </div>
-
+
```
Comment (tomratcliffe) [inline index.html]: You've got some extra whitespace here - could you remove it and perhaps turn on the auto whitespace trimming option in your editor?

File: .gitignore (hunk 1 above)
Comment (mugshepherd) [reply .gitignore]: done - thank you.

File: index.html (hunk 3 above)
Comment (mugshepherd) [reply index.html]: thank you.

File: index.html (hunk 4 above)
Comment (mugshepherd) [reply index.html]: Not sure I understand - 'team' is the old image and 'work-space-1' is the new image Ryan requested.  Let's chat when you're back at your desk.

File: package.json (hunk 5 above)
Comment (mugshepherd) [reply package.json]: nice suggestion - thank you.

File: index.html (hunk 4 above)
Comment (mugshepherd) [reply index.html]: Thanks for explaining - have deleted.

File: index.html (hunk 6 above)
Comment (mugshepherd) [reply index.html]: Thanks for clarifying (and nice suggestion).  Have deleted the white space and turned on auto white space trimming

File: index.html (hunk 2 above)
Comment (mugshepherd) [reply index.html]: ok thank you.  Will look forward to hearing back on that.

**Comment by tomratcliffe** [comment]
This looks good to go pending the URL changes which we might be able to just revert in here :+1: Cheers Toby!

**Pull request merged**